components/CTAButton.tsx

```diff
@@ -0,0 +1,17 @@
+interface ButtonProps {
+  icon?: JSX.Element;
+  text: string;
+  icon_pos?: "left" | "right";
+}
+
+const CTAButton = ({ icon, icon_pos = "right", text }: ButtonProps) => {
```

Comment:
```suggestion
const CTAButton:FC<IProps> = ({ icon, icon_pos = "right", text }) => {
```
let's follow this convention